src/server/auth/server/api_server.go

```diff
@@ -74,6 +73,8 @@ const (
 	SamlPort = 654
 )
 
+var defaultAuthConfig = authclient.AuthConfig{}
```

Comment:
It is here where I want to modify the default auth config to be 
```{
  "live_config_version": 1, // prevents RMW errors
  "id_providers": [
    {
      "name": "GitHub",
      "description": "oauth-based authentication with github.com",
      "github": {}
    }
  ]
}
``` 
right?

Reply:
yup, exactly